src/pages/signupOne.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useState } from 'react';
+//import { useNavigate } from 'react-router-dom';
+
+export default function Login() {
+   const [agreeCheck, setAgreeCheck] = useState([false, false]);
+   //const navigate = useNavigate();
+
+   const handleCheckboxChange = (index: number) => {
+      const newAgreeCheck = [...agreeCheck];
+      newAgreeCheck[index] = !newAgreeCheck[index];
+      setAgreeCheck(newAgreeCheck);
+   };
+
+   const nexthandle = () => {
```

Comment:
다른곳은 camel case로 작성해주셨는데 여긴 아니네요!
```suggestion
   const nextHandle = () => {
```